filtersrv/src/main/java/org/apache/rocketmq/filtersrv/FiltersrvStartup.java

```diff
@@ -139,7 +139,7 @@ public static FiltersrvController createController(String[] args) {
 
             Runtime.getRuntime().addShutdownHook(new ShutdownHookThread(log, new Callable() {
                 @Override
-                public Object call() throws Exception {
+                public Void call() throws Exception {
                     controller.shutdown();
                     return null;
```

Comment:
@wu-sheng ,  oh, it's my mistake, I just wrote a code and the return statement need to be kept.
As I'm not a big fan of null,  I always want to kill it if possible.